tests/test_helper/microovn.bash

```diff
@@ -501,6 +501,52 @@ function wait_ovsdb_cluster_container_leave() {
     return $rc
 }
 
+# wait_ovsdb_cluster_container_join SERVER_ID CONTROL_PATH DB_NAME TIMEOUT CONTAINER1 [CONTAINER2 ...]
```

Comment:
nit: there is a generic `wait_until` function that could be used for the wait part of this, but I won't block on this. 

Example usage:
https://github.com/canonical/microovn/pull/138/commits/d3fae453133427edeb8ff6fd6b9b568391140dee#diff-99adb05bce6de38bad928c437a3672fff44ed684f06fff6f820edcae6954d3c3R114-R116